uwblueprint/community-fridge-kw
# feat: view donation popup

## Notion ticket link

[Ticket Name](https://www.notion.so/uwblueprintexecs/Task-Board-db95cd7b93f245f78ee85e3a8a6a316d)



## Implementation description
* 

![image](https://user-images.githubusercontent.com/41586370/143721375-a9aa6829-0c4f-4b27-96cd-7da486288b4f.png)



## Steps to test
1. `node seed up` to get the data
2. Check view-donations page to see an event and click on the scheduled item



## What should reviewers focus on?
* 


## Checklist
